sql/2019/05_Third_Parties/05_01.sql

```diff
@@ -0,0 +1,23 @@
+#standardSQL
+# Percentage of pages that include at least one third party resource.
+SELECT
+  COUNT(*) as numberOfPages,
```

Comment:
For each of these queries, I think we should also split out by desktop/mobile rather than grouping together. See this conversation on the Slack channel for context: https://httparchive.slack.com/archives/CGFLV2S06/p1565208006011800

Reply:
SG, done!